src/main/java/seedu/edulog/logic/commands/MarkAllCommand.java

```diff
@@ -0,0 +1,32 @@
+package seedu.edulog.logic.commands;
+
+import seedu.edulog.commons.util.ToStringBuilder;
+import seedu.edulog.logic.commands.exceptions.CommandException;
+import seedu.edulog.model.Model;
+
+/**
+ * Mark all students, usually after attendance is taken
+ */
+public class MarkAllCommand extends Command {
+    public static final String COMMAND_WORD = "markall";
+
+    public static final String MESSAGE_MARK_ALL_SUCCESS = "Mark all successful";
+
+    public static final String MESSAGE_USAGE = "markall";
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        model.markAllStudents();
```

Comment:
I like how you ensured that only model.markAllStudents() is called so that Law of Demeter is satisfied. We should not be calling the fields of model to mark all students.